tests/tests/test-proxy.ts

```diff
@@ -0,0 +1,80 @@
+import { expect } from "chai";
+import { customWeb3Request } from "../util/providers";
+import { describeDevMoonbeam } from "../util/setup-dev-tests";
+import Keyring from "@polkadot/keyring";
+import { GENESIS_ACCOUNT_PRIVATE_KEY } from "../util/constants";
+
+function delay(ms) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+describeDevMoonbeam("Pallet proxy", (context) => {
+  it("shouldn't accept unknown proxy", async function () {
+    const keyring = new Keyring({ type: "ethereum" });
+    const genesisAccount = await keyring.addFromUri(GENESIS_ACCOUNT_PRIVATE_KEY, null, "ethereum");
+    const alice = keyring.addFromUri(
```

Comment:
is this just a random mnemonic or do you get it from somewhere?
there are a bunch of dev addresses in the readme and you can copy paste the priv key in the constants

Reply:
Used a mnemonic from the Substrate docs, I'll use a dev address instead.